services/QuillLMS/spec/models/student_problem_report_spec.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: student_problem_reports
+#
+#  id                  :bigint           not null, primary key
+#  report              :string           not null
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#  feedback_history_id :bigint
+#
+# Indexes
+#
+#  index_student_problem_reports_on_feedback_history_id  (feedback_history_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (feedback_history_id => feedback_histories.id)
+#
+describe StudentProblemReport, type: :model do
+  context 'validations' do
+    it { should belong_to(:feedback_history) }
+    it { should validate_presence_of(:feedback_history_id) }
```

Comment:
If we do add a new validation, we should cover it in the spec, too.  But up to you.